CHANGELOG.md

```diff
@@ -48,6 +48,13 @@
     - Added a new `DISPLAY_TERMS_OF_USE` environment variable to trigger the display of terms of use  
 - New scripting API features:
   - Use `WA.loadSound(): Sound` to load / play / stop a sound
+  - Use `WA.showLayer(): void` to show a layer
+  - Use `WA.hideLayer(): void` to hide a layer
+  - Use `WA.setProperty() : void` to add or change existing property of a layer
```

Comment:
the new method should be on a subobject (maybe a new one ) since the latest api refactor

Reply:
Currently working on it.